Classes/Event/EventPublisher.php

```diff
@@ -103,7 +103,7 @@ public function publishMany(string $streamName, array $events, int $expectedVers
         foreach ($rawEvents as $rawEvent) {
             $eventClassName = $this->eventTypeResolver->getEventClassNameByType($rawEvent->getType());
             $event = $this->propertyMapper->convert($rawEvent->getPayload(), $eventClassName, $configuration);
-            foreach ($this->eventListenerLocator->getListeners($rawEvent->getType()) as $listener) {
+            foreach ($this->eventListenerLocator->getSynchronousListeners($rawEvent->getType()) as $listener) {
                 call_user_func($listener, $event, $rawEvent);
             }
```

Comment:
@bwaidelich are you sure yet, you want to go the "job" way for the first implementation? Or just "ignore" asynchronous listeners at this point and only play events to them on `catchup()`?